metadata-ingestion/src/datahub/ingestion/source/metadata/business_glossary.py

```diff
@@ -148,22 +149,41 @@ def make_glossary_term_urn(
 
 def get_owners(owners: Owners) -> models.OwnershipClass:
     owners_meta: List[models.OwnerClass] = []
+
+    valid_owner_types = [
```

Comment:
There is already a function implementing that functionality - https://github.com/datahub-project/datahub/blob/master/metadata-ingestion/src/datahub/emitter/mce_builder.py#L347